host/metrics.go

```diff
@@ -0,0 +1,215 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+	"sync"
+
+	host "github.com/flynn/flynn/host/types"
+	"github.com/inconshreveable/log15"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func registerMetrics(state *State, log log15.Logger) error {
+	labels := prometheus.Labels{
+		"host": state.id,
+	}
+
+	return prometheus.Register(&metrics{
+		state: state,
+		log:   log,
+		memFreeDesc: prometheus.NewDesc(
+			"host_memory_total_bytes",
+			"Host total memory in bytes",
```

Comment:
this is called `memFreeDesc`, but says "total memory", total memory doesn't change, so this should be called `host_memory_free_bytes`, right?

Reply:
I've renamed it `memTotalDesc` and it is supposed to be the total memory so that it can be used along with available memory (`host_memory_avail_bytes`) to calculate percentage used. Available memory is the most useful metric here though.